server/modules/email/emailModule.js

```diff
@@ -16,12 +16,13 @@ const redirectUrl = process.env.GMAIL_REDIRECT_URL;
 const refreshToken = process.env.GMAIL_REFRESH_TOKEN;
 const user = process.env.EMAIL_USER;
 const opsEmail = process.env.OPS_EMAIL;
+const itEmail = process.env.IT_EMAIL;
```

Comment:
Please add this(`process.env.IT_EMAIL`) along with other emails that are set through environment variables to the `.env.sample`.

Reply:
fixed